src/snowcli/cli/nativeapp/artifacts.py

```diff
@@ -2,9 +2,14 @@
 import shutil
 from dataclasses import dataclass
 from pathlib import Path
-from typing import List, Union
+from typing import Any, List, Optional, Union
 
 from click.exceptions import ClickException
+from strictyaml import (
+    YAML,
+    as_document,
+    load,
```

Comment:
nit: I generally prefer to stick to qualified access for names like this that are highly ambiguous, for readability. I know we do this in a lot of places, but I'd be in favour of phasing those out.

Reply:
Changed it here, since `load` is indeed ambiguous.